src/jni/org_netbsd_liblpm_LPM.c

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2016 Henry Rodrick <henry at holodisc org uk>
+ * All rights reserved.
+ *
+ * Use is subject to license terms, as specified in the LICENSE file.
+ */
+
+#include <inttypes.h>
+
+#include "org_netbsd_liblpm_LPM.h"
+#include "lpm.h"
+
+void dtor(void *, const void *, size_t, void *);
+
+void dtor(void *arg, const void *key, size_t len, void *val)
```

Comment:
This should be a _static_ function. You can also remove the prototype above then.